src/rpm/builder.rs

```diff
@@ -361,7 +364,11 @@ impl RPMBuilder {
                     header_digest_sha1.as_str(),
                     header_and_content_digest_md5.as_slice(),
                 )
-                .build(header_and_content_len.try_into().unwrap())
+                .build(
+                    header_and_content_len
+                        .try_into()
+                        .expect("signature header + signature length must be <4gb"),
```

Comment:
We can do away with these once https://github.com/rpm-rs/rpm/issues/65 is handled.